components/rhel/rhel-ose/Vagrantfile

```diff
@@ -86,8 +86,10 @@ Vagrant.configure(2) do |config|
     echo "You can now access the OpenShift console on: https://${OSIP}:8443/console"
     echo
     echo "To use OpenShift CLI, run:"
-    echo "$ vagrant ssh"
-    echo "$ oc login"
+    echo "$ oc login ${OS_IP}:8443"
+    echo
+    echo "To download and install OC binary, run:"
+    echo "vagrant service-manager install-cli openshift"
```

Comment:
If I have understood this correctly, how about putting the "To download and install OC binary, run... step before the "To use OpenShift CLI, run...step?
That way a new user would first download, install and then use, and a user who already has it will simply skip downloading and installing and move on to using it.